src/main/java/org/fungover/thunder/PackageReader.java

```diff
@@ -9,7 +9,7 @@
 import java.util.Map;
 
 public class PackageReader {
-    private final Map<InetAddress, Boolean> connectPackageSent = new HashMap<>();
+    public final Map<InetAddress, Boolean> connectPackageSent = new HashMap<>();
```

Comment:
Why does this need to be public? Isn't it better to have it private and use a getter for it?

Reply:
Thank you for the feedback. you are right and we have made it private again :)